app/services/logging/logger.js

```diff
@@ -1,17 +1,143 @@
-'use strict';
+"use strict";
+
+const path = require("path");
```

Comment:
This needs to be injected as a dependency. Including "path" directly in your code turns these tests into integration tests. Path should be mocked and spied upon as appropriate.